wishlist/app/com/commercetools/sunrise/wishlist/WishlistCreator.java

```diff
@@ -0,0 +1,17 @@
+package com.commercetools.sunrise.wishlist;
+
+import com.commercetools.sunrise.wishlist.controllers.DefaultWishlistCreator;
+import com.google.inject.ImplementedBy;
+import io.sphere.sdk.shoppinglists.ShoppingList;
+
+import java.util.concurrent.CompletionStage;
+
+@ImplementedBy(DefaultWishlistCreator.class)
+public interface WishlistCreator {
```

Comment:
Could you tag it as a `@FunctionalInterface` and extend it from `ResourceFinder` and `Supplier<CompletionStage<ShoppingList>>` (this one is just to be coherent with the naming, I might remove it in the future though it doesn't hurt)? That's how all other creators/finders look like.